showcase/src/assets/translations/nl.json

```diff
@@ -331,6 +331,43 @@
         "TITLE": "Toast notification"
       }
     },
+    "DEMO_RBAC": {
+      "DIRECTIVES": {
+        "AUTHORIZATION": {
+          "HIDE_ON_PERMISSION": {
+            "TITLE": "Hide On Permission directive",
+            "SWITCH_UNAUTHORIZED_ROLE": "Schakel de niet-geautoriseerde rol:",
+            "ONLY_VISIBLE_TO": "Het is alleen zichtbaar voor een geautoriseerde gebruiker die niet de '{{ role }}' rol heeft."
+          },
+          "SHOW_ON_PERMISSION": {
+            "TITLE": "Show On Permission directive",
+            "SWITCH_AUTHORIZED_ROLE": "Schakel de geautoriseerde rol:",
+            "ONLY_VISIBLE_TO": "Het is alleen zichtbaar voor een geautoriseerde gebruiker die de '{{ role }}' rol heeft."
+          },
+          "CONTENT_PROTECTED": "Deze inhoud is beveiligd.",
+          "TITLE": "Authorization directives"
+        }
+      },
+      "SERVICES": {
+        "AUTHORIZATION": {
+          "FUNCTIONALITY": "Functionaliteit",
+          "DESCRIPTION_METHODS": "De <code>StarkRBACAuthorizationService</code> biedt de benodigde methoden om te weten of de gebruiker een specifieke rol of reeks rollen heeft en of het een anonieme gebruiker is.",
+          "DESCRIPTION_TRANSITION_HOOKS": "Het voegt ook een \"transition hook\" toe aan de router om bepaalde routes te beveiligen die toegankelijk zijn voor ongeautoriseerde gebruikers. Met de overgangshaak kan de gebruiker niet navigeren naar een route met specifieke machtigingsvereisten die op die routes zijn gedefinieerd. Bovendien kan de gebruiker worden omgeleid naar een andere route als een dergelijke omleiding wordt gedefinieerd als onderdeel van de routemachtigingsconfiguratie.",
+          "PROTECTED_ROUTES": "Beschermde routes uitdelen",
+          "DESCRIPTION_PROTECTED_ROUTES": "De beveiligde routes moeten worden gedeclareerd via de <code>StarkRBACStateDeclaration</code> interface en de configuratie van 'permissions' in het object 'data' van de routes definiëren.<br>In dit voorbeeld zijn ze als volgt gedefinieerd:",
```

Comment:
"Beschermde routes moeten gedeclareerd worden door middel van de StarkRBACStateDeclaration interface en de definitie van de 'permissions' configuratie in het data object van de route. In dit voorbeeld zijn ze als volgt gedefinieerd:"